git/operations.go

```diff
@@ -128,28 +128,27 @@ func getNotesRef(ctx context.Context, workingDir, ref string) (string, error) {
 	return strings.TrimSpace(out.String()), nil
 }
 
-func addNote(ctx context.Context, workingDir, rev, notesRef string, note *Note) error {
+func addNote(ctx context.Context, workingDir, rev, notesRef string, note interface{}) error {
 	b, err := json.Marshal(note)
 	if err != nil {
 		return err
 	}
 	return execGitCmd(ctx, workingDir, nil, "notes", "--ref", notesRef, "add", "-m", string(b), rev)
 }
 
-// NB return values (*Note, nil), (nil, error), (nil, nil)
-func getNote(ctx context.Context, workingDir, notesRef, rev string) (*Note, error) {
+// NB first return value indicates whether there was a note; second was an error
```

Comment:
Can this comment be replaced by named return values instead? `(ok bool, err error)` seems like a good choice.